scrapy/resolver.py

```diff
@@ -16,8 +16,7 @@ def __init__(self, reactor, cache_size, timeout):
     def getHostByName(self, name, timeout=None):
         if name in dnscache:
             return defer.succeed(dnscache[name])
-        if not timeout:
-            timeout = self.timeout
+        timeout = (self.timeout,)
```

Comment:
> the question is: Can DNS_TIMEOUT be at tuple?

I guess not without some other major changes at https://github.com/scrapy/scrapy/blob/4ca191eaf22eceffb75da77f9e24d5f3e87665be/scrapy/crawler.py#L290